src/demo.rs

```diff
@@ -620,9 +621,10 @@ fn handle_packet(packet: &Packet, state: &GameState) -> Vec<DemoMessage> {
             };
 
             match event {
-                // GameEvent::VoteStarted(e) => {
-                //     tracing::info!("Vote started: {:?}", e);
-                // }
+                GameEvent::VoteStarted(e) => out.push(DemoMessage {
+                    tick: tick.0,
+                    event: DemoEvent::VoteCreated(e.clone()),
+                }),
```

Comment:
Ideally this vote event does exist in the demo, and does result in events. I'm assuming the parser just fails to pull them out at this point, but I would like the SSE environment be ready to handle these events should they occur. I don't agree with not having a parsing environment set up for these events just because they aren't occuring currently, but should be

Reply:
Yes ideally we would get it, but I'm not convinced it's there as the parser seems to parse out all the packets without running into any issues and there could have been reasons for the devs to have changed how votes are initiated. I personally still don't see any reason to add handling for it unless you're totally convinced we will one day see that event, which I'm not, but I'm not going to force you to take it out or anything.

If you want to keep it then please rename the enum variants to reflect their events accurately, add a comment explaining that we should not rely on this event because we've never seen it, and also add that tracing info log back so that if we ever *do* see it then we will know.

Reply:
Ok. See [27f4890](https://github.com/MegaAntiCheat/client-backend/pull/144/commits/27f48907c92e939ef692e585d7b72b91f4ff9087)

Reply:
Cool, I'm happy with that then.